SwiftVoiceQuickstart/ViewController.swift

```diff
@@ -189,26 +194,46 @@ class ViewController: UIViewController {
         toggleAudioRoute(toSpeaker: sender.isOn)
     }
     
+    @IBAction func customAudioDeviceToggled(_ sender: UISwitch) {
+        if sender.isOn {
+            if #available(iOS 11.0, *) {
+                audioDevice = ExampleAVAudioEngineDevice()
+                TwilioVoice.audioDevice = audioDevice
+            }
+        }
+    }
+    
+    @IBAction func playMusic(_ sender: UIButton) {
+        if #available(iOS 11.0, *) {
+            if audioDevice is ExampleAVAudioEngineDevice {
+                let device = audioDevice as! ExampleAVAudioEngineDevice
```

Comment:
I think this could be:
```swift
            if let device = audioDevice as? ExampleAVAudioEngineDevice {
                device.playMusic()
            }
```
I see this casting is in a few other places. If this can be reduced to one operation and the `!` goes away I think that will be better. If that doesn't work I may have other tricks.